Source/controls/plrctrls.cpp

```diff
@@ -1181,6 +1456,9 @@ void PerformSpellAction()
 		else if (pcurs > CURSOR_HAND) {
 			TryIconCurs();
 			NewCursor(CURSOR_HAND);
+		} else {
+            CheckInvItem(true);

```

Comment:
make sure to run clang-format, there are a few broken code styles in this file

Reply:
I ended up setting it up too late in the process. I'll run it correctly on the parts I changed.